Formula/solana.rb

```diff
@@ -0,0 +1,43 @@
+class Solana < Formula
+  desc "Web-Scale Blockchain for decentralized apps and marketplaces"
+  homepage "https://solana.com"
+  url "https://github.com/solana-labs/solana/archive/refs/tags/v1.7.11.tar.gz"
+  sha256 "ed70aaed16910eab637eff0499c6e4762350c4ad50340fcb1b4bf94d23f1f97a"
+  license "Apache-2.0"
+
+  depends_on "protobuf" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "llvm" => :build
+    depends_on "make" => :build
+    depends_on "pkg-config" => :build
+
+    depends_on "openssl@1.1"
+    depends_on "systemd"
+    depends_on "zlib"
+  end
```

Comment:
```suggestion
  uses_from_macos "llvm" => :build
  uses_from_macos "zlib"

  on_linux do
    depends_on "make" => :build
    depends_on "pkg-config" => :build
    depends_on "openssl@1.1"
    depends_on "systemd"
  end
```
Are you sure you need the `depends_on "make"`? I think you can do without it.

Reply:
Good point. Build passed without Make.